app/build.gradle.kts

```diff
@@ -276,3 +277,19 @@ afterEvaluate {
         .filter { it.name.contains("Fdroid") }
         .forEach { it.enabled = false }
 }
+
+sonar {
+    properties {
+        property("sonar.projectKey", "bitwarden_android")
+        property("sonar.organization", "bitwarden")
+        property("sonar.host.url", "https://sonarcloud.io")
+        property("sonar.sources", "app/src/main/,app/src/debug/")
+        property("sonar.tests", "app/src/test/")
```

Comment:
ℹ️ Double-check these paths for me.

Reply:
The paths look correct but do we only want `main` and `debug`?

Reply:
Which others are appropriate? This came over from https://github.com/bitwarden/authenticator-android/tree/main/app/src that only has those two folders that aren't test code, but I can list out whatever else we want to include here that we consider "source" for the app itself.

Reply:
The ones @SaintPatrck added are the only ones that have actual Kotlin code in them but wouldn't we want to coder all the types and variants, it is possible that code could be added to any of them in the future.

Reply:
@david-livefront I agree but am just implementing suggestions -- if you call out additional ones I can make a quick PR, or of course you can too -- we should shoot for as much coverage as possible.

Reply:
https://github.com/bitwarden/android/pull/3349 opened this up to be more generic.